contracts/anchors/bridged/LinkableCompTokenAnchorPoseidon2.sol

```diff
@@ -25,14 +27,72 @@ contract LinkableCompTokenAnchorPoseidon2 is LinkableAnchorPoseidon2 {
     token = address(_token);
   }
   
-  function wrap(address tokenAddress, uint256 amount) public {
-    TokenWrapper(token).wrap(msg.sender, tokenAddress, amount);
+  function wrap(address tokenAddress, uint256 amount) external {
+    ITokenWrapper(token).wrap(msg.sender, tokenAddress, amount);
+  }
+  
+  function wrapAndDeposit(address tokenAddress, bytes32 _commitment) external {
+    // have to use a struct to avoid stack too deep error
+    ITokenWrapper.WrapAndDepositInput memory input = ITokenWrapper.WrapAndDepositInput(
+      {tokenAddress: tokenAddress, amount: denomination});
+    try ITokenWrapper(token).wrapAndDeposit(msg.sender, input) 
```

Comment:
We don't need to try/catch if we aren't handling the error at all.

Reply:
understood. thank you